src/main/java/seedu/address/logic/parser/parserutils/CommandTypeMatcher.java

```diff
@@ -65,13 +65,15 @@ private boolean isQuizCommand(String command) {
         String lowercaseCommand = command.toLowerCase();
         switch (lowercaseCommand) {
         case "cancel":
+        case "refresh":
         case "c":
         case "flip":
         case "w":
```

Comment:
Maybe you can leave the name of the command in the Command itself like the one in the ab3. I think that can avoid "magic string".